packages/main/src/backend/import/proxyConfig.ts

```diff
@@ -0,0 +1,111 @@
+import http from 'http';
+import https from 'https';
+import { ProxyAgent } from 'proxy-agent';
+import logger from '/@/logging/logger';
+
+// Store original global agents
```

Comment:
remove comments, most of the time we don't need them. Only on edge cases

Reply:
There is no need for them as well